plugins/lookup/github_app_access_token.py

```diff
@@ -55,8 +55,8 @@
     dest: /srv/checkout
   vars:
     github_token: >-
-      lookup('community.general.github_app_access_token', key_path='/home/to_your/key',
-             app_id='123456', installation_id='64209')
+      {{ lookup('community.general.github_app_access_token', key_path='/home/to_your/key',
+             app_id='123456', installation_id='64209') }}
```

Comment:
Before `app_id` was aligned to `'community...`, so it's better to re-align it:
```suggestion
                app_id='123456', installation_id='64209') }}
```